js/bootstrap-datepicker.js

```diff
@@ -2034,3 +2038,13 @@
 	});
 
 }));
+
+function deprecate(msg) {
```

Comment:
@Azaret do you maybe have some time soon to push an update for the `deprecated` method? Or just post here a small snippet on how we should do this and I will add it!